DMTN-248.tex

```diff
@@ -46,8 +46,245 @@
+(rotating debris in low earth orbit that appears as a row of point sources), ways to identify and flag \texttt{DiaSources}
+as potential glints in the AP pipeline are underway, instead of leaving this identification solely to brokers.
+
+Furthermore, although the multi-resolution aspect of the proposal aims to provide a larger area without significantly
+adding to the size of the alert packet, it is currently unclear, e.g., how much extra header information is added, or whether the
+FITS would need multiple extensions.
+
+\textbf{Recommendation}: At this time, do not increase the size of postage stamps, but potentially reconsider in the future.
+
+
+\section{Packet compression}\label{sec:compression}
+
+\textbf{Proposal}: Compress alert packets to reduce their size.
+
+The application of gzip compression could further reduce the size of a full alert to 65 KB (80\%).
+This might help to avoid alert distribution bottlenecks, lower the cost for brokers' storage needs, and potentially enable 
+additional full streams in the future -- all of which could positively impact time-domain science with the LSST.
+
+However, the time and computational resources required to compress the alert packets needs to be considered.
+For example, gzip compression at 50 MB/s to compress 10000 alerts could take 10000 $\times$ 0.08 MB per alert, 50 MB/s, or 
+16 seconds, which is very significant in light of the 60 second alert distribution timescale. 
+
+Furthermore, compressing the alert packets forces brokers to then decompress the alerts on arrival, which would incur further delay.
+Science goals requiring very low-latency alerts distribution might be negatively impacted by compression.
+
+Compression should only be consisdered if an algorithm could provide a compression rate of 10 MB/s, requiring only a few seconds 
+to compress a visit's worth of alerts, or if the alert distribution latency requirement is relaxed from 1 minute.
+
+\textbf{Recommendation}: Do not compress alert packets, it imposes a risk to rapid time-domain science.
+Potentially reconsider compression if it requires only a few seconds per visit.
+
+
+\section{Multiple packet formats}\label{sec:multiformats}
+
+\textbf{Proposal}: Allow brokers to specify whether their stream should have histories or postage stamps removed.
+
+The main motivation here is not scientific, but to help reduce brokers' processing costs spent removing unneeded information 
+from alert packets. 
+
+There are a few drawbacks to this proposal, but they are mostly technical:
+
+\begin{enumerate}
+\item New scope, as the current plan is for Prompt Processing to create a uniform alert packet.
+\item Streams of full-sized packets could unfairly suffer delays more often (e.g., in crowded fields).
+\item Non-identical packets might cause bookkeeping issues for downstream brokers subscribing to multiple brokers.
```

Comment:
and additional provenance work for Rubin -- we will need to manage and track which alerts were sent to which broker.

Reply:
Indeed; added ", and this would create additional provenance work for Rubin (i.e., tracking which alerts were sent to which broker)." to the first enumerated list item.